docs/intro.rst

```diff
@@ -21,18 +21,53 @@ Software developers have long known that automated testing is essential for \
 managing complex codebases. Great Expectations brings the same discipline, \
 confidence, and acceleration to data science and engineering teams.
 
+
 Why would I use Great Expectations?
 -----------------------------------
 
-To get more done with data, faster. Teams use great\_expectations to
+To get more done with data, faster. Teams use Great Expectations to
 
 -  Save time during data cleaning and munging.
 -  Accelerate ETL and data normalization.
 -  Streamline analyst-to-engineer handoffs.
+-  Streamline knowledge capture and requirements gathering from subject-matter experts.
 -  Monitor data quality in production data pipelines and data products.
+-  Automate verification of new data deliveries from vendors and other teams.
 -  Simplify debugging data pipelines if (when) they break.
--  Codify assumptions used to build models when sharing with distributed
-   teams or other analysts.
+-  Codify assumptions used to build models when sharing with other
+   teams or analysts.
+-  Develop rich, shared data documention in the course of normal work.
+-  Make implicit knowledge explicit.
+-  ... and many more
+
+
+Key features
+--------------------------------------------------
+
+**Expectations**
+
+	Expectations are the workhorse abstraction in Great Expectations. Like assertions in traditional python unit tests, Expectations provide a flexible, declarative language for describing expected behavior. Unlike traditional unit tests, Great Expectations applies Expectations to data instead of code.
+
+	Great Expectations currently supports native execution of Expectations in three environments: pandas, SQL (through the SQLAlchemy core), and Spark. This approach follows the philosophy of "take the compute to the data." Future releases of Great Expectations will extend this functionality to other frameworks, such as dask and BigQuery.
+
+**Automated data profiling**
+
+	Writing pipeline test from scratch can be tedious and counterintuitive. Great Expectations jump starts the process by providing powerful tools for automated data profiling. This provides the double benefit of helping you explore data faster, and capturing knowledge for future documentation and testing.
+
+**DataContexts and DataSources**
+
+	...allow you to configure connections your data stores, using names you’re already familiar with: “the ml_training_results bucket in S3,” “the Users table in Redshift.” Great Expectations provides convenience libraries to introspect most common data stores (Ex: SQL databases, data directories and S3 buckets.) We are also working to integrate with pipeline execution frameworks (Ex: airflow, dbt, dagster, prefect.io). The Great Expectations framework lets you fetch, validate, profile, and document your data in a way that’s meaningful within your existing infrastructure and work environment.
```

Comment:
Is the "..." intentional?

Reply:
yes, since it's a continuation of the header.